indico/modules/events/surveys/controllers/management/survey.py

```diff
@@ -163,3 +169,67 @@ def _send_emails(self, form, recipients):
             email = make_email(to_list=recipient, bcc_list=bcc, from_address=form.from_address.data,
                                template=tpl, html=True)
             send_email(email, self.event, 'Surveys')
+
+
+class RHEmailEventSurveyBase(RHManageSurveyBase):
+    """Send emails to event participants."""
+
+    def _process_args(self):
+        RHManageSurveyBase._process_args(self)
+        registrations = Registration.get_all_for_event(self.event)
+        self.recipients = {x.email for x in registrations}
+
+
+class RHAPIEmailEventSurveyMetadata(RHEmailEventSurveyBase):
+    def _process(self):
+        with self.event.force_event_locale():
+            tpl = get_template_module('events/surveys/emails/survey_link_email.html', event=self.event)
```

Comment:
This test is quite misleading when the survey is NOT open. Maybe disable the button in that case...